### PR TITLE
@ #363 | should update releases/vX.X.X convention

### LINKED ISSUE #363: should update releases/vX.X.X convention
instead of releases/X.X.X on http://dev.teracy.org/docs/release_process.html

We're using vX.X.X convention for all of our version numbers.

### PR DESCRIPTION
**Pull request by hoatle**
connect #363

### CODE REVIEW

File: docs/release_process.rst [hunk 1]
```diff
@@ -6,8 +6,12 @@ This is the general guide for releasing software packages at Teracy.
 Starting
 --------
 
-- Follow workflow, a release branch should be branched off from origin branch that is ready to be
-  released (for example: ``releases/0.1.0`` branch is branched off from the ``develop`` branch).
+- Follow workflow, a releasing branch should be branched off from origin branch that is ready to be
```
Comment (hoavt) [inline docs/release_process.rst]: --> from an origin branch

File: docs/release_process.rst [hunk 2]
```diff
@@ -6,8 +6,12 @@ This is the general guide for releasing software packages at Teracy.
...
+  released.
+
+- The releasing branch should be named ``releases/vX.X.X``.
+
+- For example: ``releases/v0.1.0`` branch is branched off from the ``develop`` branch.
```
Comment (hoavt) [inline docs/release_process.rst]: For example: The ....

File: docs/release_process.rst [hunk 3]
```diff
@@ -39,7 +43,7 @@ Ending
   used for auto deployment, so make sure the message has the correct pattern.
 
 - Merge the tag release into the ``master`` branch, then the origin branch. This is specified on the
-  workflow.
+  project workflow.
 
 - Delete released branch and we are done.
```
Comment (hoavt) [inline docs/release_process.rst]: Delete the released branch...

File: docs/release_process.rst [hunk 4]
```diff
@@ -26,7 +30,7 @@ Releasing
 - Remember that release branch should not introduce big changes.
 
 - After each staging releases, create an issue for release test campaign to make sure the release
```
Comment (hoavt) [inline docs/release_process.rst]: After each staging release,

**Review by hoavt — COMMENTED** [review]
@hoale: Please check my comments.

**Comment by hoatle** [comment]
@hoavt feedbacks applied.

**Review by hoavt — APPROVED** [review]

**Pull request merged**